Makefile

```diff
@@ -9,9 +9,17 @@ install:
 test:
 	nosetests $(TEST_PATH)
 
+
 jupyter_server:
 	jupyter notebook $(NOTEBOOK_PATH)
 
+sync_notebooks:
+	jupytext --sync $(NOTEBOOK_PATH)/*.ipynb
+
+check_notebooks_synced:
+	jupytext --test-strict -x $(NOTEBOOK_PATH)/*.ipynb --to py:percent
```

Comment:
how about also doing

```
format_notebooks:
	jupytext --pipe 'isort - --treat-comment-as-code "# %%" --float-to-top' $(NOTEBOOK_PATH)/*.ipynb
	jupytext --sync --pipe black $(NOTEBOOK_PATH)/*.ipynb
```
This runs black and isort on the notebooks which is pretty cool

Reply:
Yeah, I like that! I'll do that as part of enabling black, isort and lint on the whole repo if that's OK.